bin/civihr-install

```diff
@@ -633,6 +650,12 @@ function civihr_parse_options() {
         shift
         ;;
 
+      --force-drop)
+        mysql -u$DB_USER $PASS_PARAM -e "drop database $CMSdbName" || { echo "[[Please fix above MySQL error.]]"; exit 1;  }
+        mysql -u$DB_USER $PASS_PARAM -e "drop database $CIVIdbName" || { echo "[[Please fix above MySQL error.]]"; exit 1;  }
```

Comment:
I don't its a good idea to actually execute an action while parsing the options. We should just set a flag if the option is present and, probably on the functions which install the databases, drop them if the flag is set.

Putting the drop command here, also make it impossible to add the `--force-drop` option before the `--dbpass` param, because it depends on the $PASS_PARAM variable which is only set when the `--dbpass` option is parsed. Try this to reproduce this issue:

- Create a new site using a db user with a password
- After the site is created, use the same command from the previous step, but add the `--force-drop` option **BEFORE** the `--dbpass` option

The installation will fail saying it was not possible to connect to the database without a password, even though you passed the password as a param to the script